NEWS.md

```diff
@@ -3,6 +3,15 @@ ergw-aaa
 
 Erlang AAA session implementation for ERGW.
 
+Version 3.6.9 - 4 February 2021
+----------------------------
+
+**Features** :rocket:
+* [#116](https://github.com/travelping/ergw_aaa/pull/116) Adding the `DIAMETER` `RFC` base dictionary to all services
```

Comment:
what does this yield in terms of features?

Reply:
Looks like this is **Features** by reason that - we was not have `DIAMETER` `RFC` base in project before, if we add something new this is **features**, if we change something for eg speed up reload something etc. - this is **improvements**. Plus for current implementation was created branch with prefix `feature`.